src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/FrozenStringNode.cs

```diff
@@ -2,7 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Collections.Generic;
+using System.Text;
```

Comment:
These `usings` should not be needed because of nothing else changed in this file